### DISCUSSION TITLE
Reunión GArCyAR 2023

### DISCUSSION THREAD

**Discussion by HectorLestani** [GArCyAR/ReunionesAnuales#1] [Ideas]
## Y la reunión del año próximo?
Estamos buscando una/s persona/s que se haga/n cargo de organizar la reunión GArCyAR 2023.
Esto incluye definir lugar, fecha, etc.
En este espacio podemos discutir al respecto, adelante.

**Comment by gtheler** [comment]
(mensaje de Martín Silva)

Estimados,

 

Perdón por la demora. Vengo con los resultados de la encuesta para la reunión de este año.

 

En líneas generales (abajo un detalle) parece haber preferencia por la modalidad presencial en NASA-Nodus (Villa Martelli), a favor de hacer visitas en Atucha, alrededor de la última semana de noviembre. Tendríamos 10 expositores y podríamos hacer alguna mesa redonda sobre estado del arte de metodologías de cálculo, colaboración y desafíos a futuro.

 

Estoy intentando averiguar qué pasa con la AATN este año, para que no interfieran las fechas, y tengo que asegurar la disponibilidad de lugar para reunirnos en Nodus y visitar Atucha/CAREM. Ahí el tema principal es la logística de veintipico de personas (si se puede conseguir micro). Es probable que no tengamos suerte con el Simulador así que apuntaría a que sean 2 días.

 

Se acepta todo tipo de sugerencia u observaciones.

Saludos neutrónicos.

Martín

 

 

El detalle de lo votado.

---------------------------

 

Votaron 26 personas y algunos más me expresaron sus opiniones verbalmente.

El universo de votantes fue bastante representativo de los ámbitos que habitamos, aunque no hubo participantes de la ARN.

 

En cuanto a la modalidad de reunión, 46% votó por presencial, 35% por híbrido y 19% por videoconferencia (preferentemente por G Meet, 68% vs. Zoom 27%).

 

La propuesta de hacer visitas en el predio de Atucha/Carem tuvo alta aceptación ya que sólo dos votantes no seleccionaron opciones con visita.

De los que expresaron sus preferencias, a 14 les interesa vistar el simulador de Atucha y a a 17 Atucha/Carem.

 

En cuanto a las fechas, la mayoría apunto a la segunda quincena de noviembre o primera de diciembre.

 

Tenemos tentativamente 10 expositores y se mencionaron 4 temas para discusión general:

-  Línea de cálculo para PWR
-  Actualizacion en Metodologias de Calculo en codigos deterministicos + Actualizacion de Bibliotecas - Analizar estado actual y desafios a futuro
-  (Cooperación en) desarrollo y validación de códigos neutrónicos de CNEA-NASA-INVAP
- Uso de inteligencia artificial (LLM) en la industria nuclear

 

Se propusieron, además, talleres de cálculo neutrónico en GPU y Diseño de reactores PWR

**Comment by martinsebastiansilva** [comment]
Estimados,

Les pido que vayan enviando los títulos (y abstract/texto completo si lo hubiere) de los trabajos que quieran presentar y si tienen alguna preferencia en cuanto a fecha/horario/modo de exposición (presencial/virtual), así empiezo a organizar la reunión. Recibiré los trabajos en mi correo de na-sa ([msilva@na-sa.com.ar](mailto:msilva@na-sa.com.ar)).

La reunión será entre el **5 y el 7 de diciembre**. Este último día se hará la visita a Atucha y Carem. Tengo que averiguar si vamos a poder tener disponibilidad en el simulador de Atucha el 5 o el 6 en ese caso se podrían hacer lagunas charlas allá. O si les parece que es mucho, lo dejamos para la próxima reunión para no tener que viajar 2 veces.

Tenemos tiempo para ir armando pero cuanto antes vayan enviando me dan más margen para coordinar situaciones especiales (aquellos que vengan un solo día o expongan en forma remota).
Saludos a todos,

**Comment by martinsebastiansilva** [comment]
Buenos días, nos vamos acercando a la reunión. Nos queda menos de un mes.
Les paso el formulario de inscripción para relevar vuestros datos y poder realizar las gestiones administrativas en NASA.

https://forms.gle/SE5L5Yy3VnguzTce9

Favor de completarlo antes del lunes 13/11. Lleva dos minutos. Extenderlo a todos los que no estén en los medios de comunicación del grupo.

 Los que participarán en modo virtual no necesitan llenarlo excepto que presenten trabajos, en cuyo caso les pido que incluyan el título así puedo armar el cronograma preliminar.
